blockchains/elrond/assets/WEGLD-bd4d79/info.json

```
@@ -0,0 +1,48 @@
+{
+    "type": "ESDT",
+    "id": "WEGLD-bd4d79",
+    "name": "WrappedEGLD",
+    "symbol": "WEGLD",
```

```suggestion
    "symbol": "wEGLD",
```

is it `wEGLD` or `WEGLD` ?

---

It's `WEGLD` - I've fixed the description.